### PR TITLE
update acala types

### PR DESCRIPTION
**Pull request by qwer951123**

### CODE REVIEW

**Comment by jacogr** [comment]
Not sure why the build is failing, all the pieces looks to be there. Will click through and look at the logs.

File: packages/apps-config/src/api/spec/index.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ const spec: Record<string, OverrideBundleDefinition> = {
   Pangolin: pangolin,
   VLN: vln,
   'VLN-PC': vlnrococo,
-  acala,
+  ...acala,
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/index.ts:100]: The issue is that above you export an `OverrideBundleDefinition` and here you expand it. This is why the build fails.

Comment (qwer951123) [reply packages/apps-config/src/api/spec/index.ts:100]: fixed the type assertion

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**